### ISSUE TITLE
Redesign of about page

### ISSUE THREAD

**Issue by Panosfunk** [cph-cachet/carp_studies_app#330]
https://www.figma.com/design/zrsrJzIoL5Zn4S6aijwc3a/CARP-Study-App?node-id=2765-3297&node-type=frame&t=d7nLhgHCZWXsRL03-0

**Comment by Panosfunk** [comment]
@Panosfunk make sth better for status card error message.

**Comment by Panosfunk** [comment]
Check 'contact researcher', 'study responsible', 'participant role', 'device role' and compare with running app

**Comment by Panosfunk** [comment]
this is correct

**Comment by Panosfunk** [comment]
> [@Panosfunk](https://github.com/Panosfunk) make sth better for status card error message.

Done

**Comment by Panosfunk** [comment]
@bardram This cannot be done. Protocols dont fully support registration of two primary devices assigned to two different roles. Also core only supports the device deployment status for only 1 primary device.
```
/// The [DeviceDeploymentStatus] for the primary device of this deployment,
  /// which is typically this phone.
  ///
  /// Returns `null` if there is no primary device in the list of [deviceStatusList].
  DeviceDeploymentStatus? get primaryDeviceStatus {
    for (DeviceDeploymentStatus status in deviceStatusList) {
      if (status.device is PrimaryDeviceConfiguration) return status;
    }
    return null;
  }
```

That does not allow me to check for the status of other primary devices.

**Comment by bardram** [comment]
This is because you're looking at the wrong service/endpoint.

You should ask the DeploymentService about the status from the server. The API / endpoint is here

* https://pub.dev/documentation/carp_core/latest/deployment_carp_core_deployment/DeploymentService/getStudyDeploymentStatus.html

You should ask the CAWS Deployment Service.

You will get back a [`StudyDeploymentStatus`](https://pub.dev/documentation/carp_core/latest/deployment_carp_core_deployment/StudyDeploymentStatus-class.html) which contains a list of `deviceStatusList` for each device, incl. the primary devices.

**Comment by Panosfunk** [comment]
Yes, this is what I did. It's also only happening when I have 2 participant roles with 2 different primary devices, but it works fine in other cases. The problem might be somewhere else in the configuration of 2 participant roles with 2 devices.
